types/taiko/index.d.ts

```diff
@@ -0,0 +1,366 @@
+    // https://taiko.gauge.org/#clearpermissionoverrides
+    export function clearPermissionOverrides(): Promise<void>;
+    // https://taiko.gauge.org/#setcookie
+    export function setCookie(name: string, value: string, options?: CookieDetailOptions): Promise<void>;
+    // https://taiko.gauge.org/#deletecookies
+    export function deleteCookies(cookieName?: string, options?: CookieOptions): Promise<void>;
+    // https://taiko.gauge.org/#getcookies
+    export function getCookies(options?: { urls: string[] }): Promise<Object[]>;
+    // https://taiko.gauge.org/#setlocation
+    export function setLocation(options: LocationOptions): Promise<void>;
+
+    /**
+     * Page Actions
+     */
+
+    // https://taiko.gauge.org/#goto
+    export function goto(url: string, options?: NavigationOptions): Promise<void>;
+    // https://taiko.gauge.org/#reload
+    export function reload(url: string, options?: NavigationOptions): Promise<void>;
+    // https://taiko.gauge.org/#goback
+    export function goBack(options?: NavigationOptions): Promise<void>;
+    // https://taiko.gauge.org/#goforward
+    export function goForward(options?: NavigationOptions): Promise<void>;
+    // https://taiko.gauge.org/#title
+    export function title(): Promise<string>;
+    // https://taiko.gauge.org/#click
+    export function click(selector: SearchElement | MouseCoordinates, options?: ClickOptions, args?: string[]): Promise<void>;
+    // https://taiko.gauge.org/#doubleclick
+    export function doubleClick(selector: SearchElement | MouseCoordinates, options?: BasicNavigationOptions, args?: string[]): Promise<void>;
+    // https://taiko.gauge.org/#rightclick
+    export function rightClick(selector: SearchElement | MouseCoordinates, options?: BasicNavigationOptions, args?: string[]): Promise<void>;
+    // https://taiko.gauge.org/#draganddrop
+    export function dragAndDrop(
+        source: SearchElement,
+        destination: SearchElement,
+        distance: DragAndDropDistance
+    ): Promise<void>;
+    // https://taiko.gauge.org/#hover
+    export function hover(selector: SearchElement, options?: EventOptions): Promise<void>;
+    // https://taiko.gauge.org/#focus
+    export function focus(selector: SearchElement, options?: EventOptions): Promise<void>;
+    // https://taiko.gauge.org/#write
+    export function write(text: string, into?: SearchElement, options?: WriteOptions): Promise<void>;
+    // https://taiko.gauge.org/#clear
+    export function clear(selector: SearchElement, options?: NavigationOptions): Promise<void>;
+    // https://taiko.gauge.org/#attach
+    export function attach(filepath: string, to: SearchElement): Promise<void>;
+    // https://taiko.gauge.org/#press
+    export function press(keys: string | string[], options?: KeyOptions): Promise<void>;
+    // https://taiko.gauge.org/#highlight
+    export function highlight(selector: SearchElement): Promise<void>;
```

Comment:
takes ...args rest parameter which is proximityElementSelector[]

Reply:
changed